bugs_and_glitches.md

```diff
@@ -399,3 +413,10 @@ HandleAIShift: ; 22476 (8:6476)
 	ret
  	...
 ```
+## Challenge host uses wrong name for the first rival
+When playing the challenge cup, player name is used instead of rival name before the first fight, as seen here: https://www.youtube.com/watch?v=1igDbNxRfUw&t=17310s
+**Fix:** Edit `Text0533` in `text6.asm`: 
```

Comment:
I'd put an extra blank line before "Fix" so that it actually appears on a new line.